app/views/stay/payments/new.html.erb

```diff
@@ -0,0 +1,314 @@
+                        <!-- Options will be populated by JavaScript -->
+                        </select>
+                    </div>
+                </div>
+                <div class="mb-3">
+                    <div class="form-group">
+                        <label for="card-element">Card Details</label>
+                        <div id="card-element"><!-- A Stripe Element will be inserted here. --></div>
+                        <div id="card-errors" class="error"></div>
+                    </div>
+                </div>
+                  <input type="hidden" id="confirm" name="confirm" value=true />
+                  <input type="hidden" id="redirect_url" name="redirect_url" value="<%= API_CONFIG['host_url'] + root_path %>" />
+                <button type="submit">Pay</button>
+            <%end%>
+            </div>
+        </div>  
+    </div>
+    
+    <div class="col-md-5">
+        <div class="card">
+          <div class="card-body">
+            <% @booking.line_items.each do |item| %>
+                <h5 class="card-title"><%= item.room.room_type.name %></h5>
+                <p class="card-text">Room in <%=@property.address.city.name%></p>
+                <p class="card-text">
+                    <small class="text-muted">
+                        <span class="fa fa-star"></span> 4.65 (190 reviews) • Superhost
+                    </small>
+                </p>
+                
+                <h3 class="mt-4">Price details</h3>
+                <div class="d-flex justify-content-between mb-2">
+                    <span>₹<%= item.room.price_per_night%> x <%=(@booking.check_out_date - @booking.check_in_date).to_i%> nights</span>
+                    <span>₹<%= item.room.price_per_night * (@booking.check_out_date - @booking.check_in_date).to_i%></span>
+                </div>
+                <div class="d-flex justify-content-between mb-2">
+                    <span>Cleaning fee</span>
+                    <span>₹0.0</span>
+                </div>
+                <div class="d-flex justify-content-between mb-2">
+                    <span>Taxes</span>
+                    <span>₹0.0</span>
+                </div>
+                <div class="d-flex justify-content-between">
+                  <strong>Total (INR)</strong>
+                  <strong>₹<%= item.room.price_per_night * (@booking.check_out_date - @booking.check_in_date).to_i%></strong>
+                </div>
+                <hr>
+              <%end%>
+              <h3 class="mt-4">Total Booking Price</h3>
+              <div class="d-flex justify-content-between">
+                <strong><%=@booking.line_items.count%> rooms X <%= (@booking.check_out_date - @booking.check_in_date).to_i%> nights</strong>
+                <strong>₹<%= @booking.total%></strong>
+              </div>
+
+            </div>
+        </div>
+    </div>
+</div>
+</div></body>
+<!-- Stripe.js library -->
+<script src="https://js.stripe.com/v3/"></script>
+
+<script>
```

Comment:
remove the js from this view file and move into payment.js file and include this file here .